audius-cli

```diff
@@ -86,11 +86,14 @@ def _is_creator_node_healthy(health_check_resp):
     return health_check_resp["data"]["healthy"]
 
 
-def _is_discovery_provider_healthy(health_check_resp):
-    return (
-        health_check_resp["data"]["block_difference"]
-        < health_check_resp["data"]["maximum_healthy_block_difference"]
-    )
+def _is_discovery_provider_healthy(health_check_resp, strict):
+    if strict:
+        return (
+            health_check_resp["data"]["block_difference"]
+            < health_check_resp["data"]["maximum_healthy_block_difference"]
+        )
+    else:
```

Comment:
no need for the else and also, this check seems a bit weird. would it be better to just return HTTP status?

Reply:
Will follow up on these changes in another PR to audius-k8s